### DISCUSSION TITLE
when increase max level to 90 lfg is not showing the ICC dungeons and raids

### DISCUSSION THREAD

**Discussion by ralver** [azerothcore/azerothcore-wotlk#18506] [Guides & Tips]
Hi there,

I increased the max level of a character to 90, but now it will not show any dungeons are raids at a level higher then 80. Is there an option or database update to het this working?

With regards Ralph.

**Comment by TheSCREWEDSoftware** [comment]
Mostly likely requires client patching, better asked in the modding commnuity
